test/expect-tests/digest/dune

```diff
@@ -0,0 +1,14 @@
+(library
```

Comment:
I don't understand: why did you have to add this file?

Reply:
We have convention for tests to have their own inline tests stanza for testing. You gain better concurrency and less test re-running this way.

Reply:
Ah, I missed the move of the `digest_tests.ml`, now it makes more sense.

I don't understand though what you mean by better concurrency/incrementality, i.e. compared to what?

Reply:
Previously, we'd have one `(inline_tests ..)` library for both stdune and digest, so those tests would run serially and have the same runtime dependencies. Now we have two test binaries that can run concurrently and the digest tests have less runtime deps.

Reply:
Ah, sure.